src/store/scenes/listener.js

```diff
@@ -59,7 +61,9 @@ const handleSceneDelete = (action, store) => {
 
   // Delete sketches
   scene.sketchIds.forEach(sketchId => {
-    store.dispatch(uSketchDelete(sketchId, p.id))
+    // We're skipping postprocessing reset to stop it happening multiple times per sketch
+    // Instead it is called once after scene is deleted
+    store.dispatch(uSketchDelete(sketchId, p.id, { skipPostProcessingReset: true }))
```

Comment:
maybe changing this so the reset function schedules a clean, and if a clean is already scheduled it doesn't do anything

Reply:
I see what you mean, although your approach may take a bit of refactoring. Happy for you to go ahead and make this change if you feel it's tidier. :)